scripts/generate_skus.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/ottogroup/penelope/pkg/processor"
+	"google.golang.org/api/cloudbilling/v1"
+)
+
+func main() {
+	ctx := context.Background()
+	cloudbillingService, err := cloudbilling.NewService(ctx)
+	if err != nil {
+		panic(err)
+	}
+
+	res, err := cloudbillingService.Services.Skus.List("services/95FF-2EF5-5EA1").CurrencyCode("EUR").Do()
+	if err != nil {
+		panic(err)
+	}
+
+	regions := map[string]processor.Location{
+		"europe-west1":      {Latitude: 50.4841084, Longitude: 3.7646014},
+		"europe-west3":      {Latitude: 50.1177289, Longitude: 8.595807},
+		"europe-west4":      {Latitude: 53.4342611, Longitude: 6.8033996},
+		"europe-west8":      {Latitude: 45.4678811, Longitude: 9.1320676},
+		"europe-west9":      {Latitude: 48.8602937, Longitude: 2.2821444},
+		"europe-west10":     {Latitude: 52.510282, Longitude: 13.2942595},
+		"europe-west12":     {Latitude: 45.0711719, Longitude: 7.633703},
+		"europe-central2":   {Latitude: 52.2300896, Longitude: 20.9108091},
+		"europe-southwest1": {Latitude: 40.440189, Longitude: -3.8079513},
+		"europe-north1":     {Latitude: 60.5695263, Longitude: 27.1520617},
+	}
+
+	classes := []string{
+		"regional",
+		"nearline",
+		"coldline",
+		"archive",
+	}
+
+	for region, location := range regions {
+		fmt.Printf("{\n")
```

Comment:
**note**: get file path explicit or with default value and then instead of `Printf` use `Fprintf` to avoid clash of generate errors with the data generation

Reply:
Current process is not to write the file directly, instead just print the data structure as go code and than manually update go file. Updating the will not happend so often(at most once a year).